scripts/deleteSongsFromPlaylist.ts

```diff
@@ -0,0 +1,48 @@
+import { Prisma } from "../generated/prisma";
+import gql from "graphql-tag";
+
+const prisma = new Prisma({
+  endpoint: "http://localhost:4466"
+});
+
+const name = process.argv[process.argv.length - 1];
+
+prisma.query
+  .playlists(
+    { where: { name } },
+    gql`
+      {
+        tracks {
+          id
+          info {
+            id
+          }
+        }
+      }
+    `
+  )
+  .then(([{ tracks }]) => ({
```

Comment:
You should be able to use async/await instead of promises, right?
```javascript
const [{tracks}] = await prisma.query.playlists( ... );
..
await prisma.mutation.deleteManyTrackInfoes({ ... });
await prisma.mutation.deleteManyTracks({ ... });
```